src/js/utils/date-utils.js

```diff
@@ -52,39 +43,71 @@ var DATE_FORMATS = {
         'jquery',
         'edx-ui-toolkit/js/utils/string-utils',
         'moment',
-        'momenttimezone'
+        'moment-timezone'
     ], function(
         $,
         StringUtils,
         moment,
         momentTZ
         ) {
-        var DateUtils, isValid, dateObject, userTimezone, localizeTime, timeDisplay, localize;
-
+        var DateUtils,
+            localize,
+            stringtoObject,
+            localizeTime,
+            _getSysTimezone,
+            getDisplayTime,
+            _getDisplayLang,
+            _getDisplayFormat,
+            _isValid;
         /**
-         * Helper function to determine variable/context validity.
          *
-         * @param {string} candidateVariable to test
-         * @returns {boolean}
+         * Renders a localized date/time in a standard format.
+         *
+         * This should take a context dict, with an additional (and optional)
+         * handler for a modifier_string (e.g. 'due', 'deadline:', etc.) via
+         * StringUtils.
+         *
+         * Please Note: the preferred method is via underscore in the mako
+         * templates, however this will also work.
+         *
+         * @param context
+         * @returns {string} A formatted, timezone-offset, internationalized date and time.
+         *
+         * - datetime: {string} The (UTC) datetime server-side string.
```

Comment:
I think these lines are describing the context, so should be above the ``@returns`` line.